sphinx/source/docs/dev_guide/documentation.rst

```diff
@@ -44,119 +109,129 @@ following command at a Bash or Windows prompt:
+``GOOGLE_API_KEY``. The documentation includes some plots with maps, and a valid
+Google API key is required to build those plots correctly. You have two
+options:
 
-.. code-block:: sh
+* Follow the instructions on the `Google developers website`_ to generate a new
+  API key.
 
-    cd sphinx
+* Use a placeholder value like ``some_value`` instead of a valid API key. If
+  you use a placeholder, some map plots in Bokeh's documentation might not be
+  rendered correctly, but the documentation should otherwise be built correctly.
 
-Sphinx uses the ``make`` tool to control the build process. The most common
-targets of the Bokeh makefile are:
+On Linux or macOS, use the following command to set the environment variable:
 
-``clean``
-    Remove all previously built documentation output. All outputs will
-    be generated from scratch on the next build.
+.. code-block:: sh
 
-``html``
-    Build any HTML output that is not built, or needs re-building (e.g.
-    because the input source file has changed since the last build).
+    GOOGLE_API_KEY=some_value
 
-``serve``
-    Start a server to serve the docs and open a web browser to display them.
-    Note that due to the JavaScript files involved, starting a real server is
-    necessary to view many portions of the docs fully.
+With the Windows PowerShell, use this command:
 
-For example, to clean the docs build directory, run the following command at
-the command line:
+.. code-block:: PowerShell
 
-.. code-block:: sh
+    $env:GOOGLE_API_KEY=some_value
 
-    make clean
+In a Windows terminal, use this command:
 
-Multiple targets may be combined in a single `make` invocation. For instance,
-executing the following command will clean the docs build, generate all HTML
-docs from scratch, and then open a browser to display the results:
+.. code-block:: doscon
 
-.. code-block:: sh
+    set GOOGLE_API_KEY=some_value
 
-    make clean html serve
+Building Bokeh's documentation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-By default, built docs will load the most recent BokehJS from CDN. If you are
-making local changes to the BokehJS and wish to have the docs use your locally
-built BokehJS instead, you can set the environment variable ``BOKEH_DOCS_CDN``
-before calling ``make``:
+You can find all source files for Bokeh's documentation in the ``sphinx``
+directory of the Bokeh source tree.
 
 .. code-block:: sh
 
-    BOKEH_DOCS_CDN=local make clean html serve
+    cd sphinx
 
-Building the docs also requires setting up the ``GOOGLE_API_KEY`` environment variable that ``gmap`` plots use.
-You can `create a new API Key <https://developers.google.com/maps/documentation/javascript/get-api-key>`_.
-Or if you don't care whether ``gmap`` plots work, you can set the variable with a fake value before calling ``make``:
+Sphinx uses the standard Unix ``make`` command to control the build process. For
+Windows users, the ``sphinx`` directory includes the file ``make.bat``. Use this
```

Comment:
@hyles-lineata I remember you did some more in-depth testing on Windows with make, make.bat and Powershell(?) - does this paragraph make sense, based on your findings?